easybuild/easyconfigs/l/libcifpp/libcifpp-7.0.8-GCC-13.3.0.eb

```diff
@@ -0,0 +1,45 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2025/01
+easyblock = 'CMakeMake'
+
+name = 'libcifpp'
+version = '7.0.8'
+
+homepage = 'https://github.com/PDB-REDO/libcifpp'
+description = """This library contains code to work with mmCIF and PDB files"""
+
+toolchain = {'name': 'GCC', 'version': '13.3.0'}
+
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = []
+checksums = ['2297e6649a4f71caf9da5f1d97f59512e7324bb62083bb5b08eb00c1c0385cb3']
+
+github_account = 'PDB-REDO'
+
+builddependencies = [
+    ('CMake', '3.29.3'),
+    ('Eigen', '3.4.0'),
+    ('Boost', '1.85.0'),  # req. for tests
+    ('mrc', '1.3.13'),
+    ('Catch2', '2.13.10')
+]
+dependencies = [
+    ('zlib', '1.3.1'),
+]
+
+configopts = [
+    '-DBUILD_SHARED_LIBS=OFF -DDENABLE_TESTING=ON -DFETCHCONTENT_FULLY_DISCONNECTED=ON ',
+    '-DBUILD_SHARED_LIBS=ON -DDENABLE_TESTING=ON -DFETCHCONTENT_FULLY_DISCONNECTED=ON ',
```

Comment:
i noticed that `-DDENABLE_TESTING` is probably a typo, but then i didn't find any variable that enables testing. instead, the readme says:
>Tests are created by default, and to test the code you can run:

```console
ctest --test-dir build
```